requirements.txt

```diff
@@ -86,7 +86,7 @@ pinax-ratings==4.0.0
 
 # GeoNode org maintained apps.
 # django-geonode-mapstore-client>=2.1.1
--e git+https://github.com/GeoNode/geonode-mapstore-client.git@master#egg=django_geonode_mapstore_client
+-e git+https://github.com/GeoNode/geonode-mapstore-client.git@mapstore_refactor#egg=django_geonode_mapstore_client
```

Comment:
Let's first merge the changes on `geonode-mapstore-client` and revert this to `master`.